sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/DescribeTableSchemas.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.catalyst.plans
+
+import org.apache.spark.sql.catalyst.expressions.AttributeReference
+import org.apache.spark.sql.types.{MetadataBuilder, StringType, StructField, StructType}
+
+private[sql] object DescribeTableSchemas {
+  val DESCRIBE_TABLE_ATTRIBUTES = Seq(
```

Comment:
We shouldn't define attributes in an object. `AttributeReference` will be assigned a unique ID when created, and in general we should create new attributes when creating a new logical plan.

I'm not sure what will be broken, but this is not a recommended pattern.

I would just define a schema here, and when I need attributes, just call `schema.toAttributes`, which creates new `AttributeReference`

Reply:
or, we call follow `DescribeCommandBase`: create attributes in an abstract class instead of object.

Reply:
Can you join the results of `DESCRIBE`?

Reply:
```
scala> val df1 = sql("desc t1")
df1: org.apache.spark.sql.DataFrame = [col_name: string, data_type: string ... 1 more field]

scala> val df2 = sql("desc t2")
df2: org.apache.spark.sql.DataFrame = [col_name: string, data_type: string ... 1 more field]

scala> df1.crossJoin(df2).show
+--------+---------+-------+--------+---------+-------+
|col_name|data_type|comment|col_name|data_type|comment|
+--------+---------+-------+--------+---------+-------+
|       i|      int|   null|       j|      int|   null|
+--------+---------+-------+--------+---------+-------+
```

This is not a common use case but we don't have to break it.

Reply:
Changed this from a value to a method, so it will generate new identifiers every time while still being shared amongst multiple contexts.